core/src/main/java/com/adobe/aio/util/JacksonUtil.java

```diff
@@ -35,10 +33,6 @@ private JacksonUtil() {
       JsonMapper.builder()
           .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
           .serializationInclusion(Include.NON_NULL)
-          .addModule(new SimpleModule()
-              .addSerializer(ZonedDateTime.class, new ZonedDateTimeSerializer())
-              .addDeserializer(ZonedDateTime.class, new ZonedDateTimeDeserializer()))
```

Comment:
How do we make sure that it won't fail in serialization/deserialization? Does Jackson support these date formats OOTB?

Reply:
what matters here is the objectmapper pulled from here is properly serialization / deserializing date when using the model stuffed in our sdk, and I believe it's the case : the only 2 dates found in our json model are actually currently exposed as String in the `Registration` class : they were no need for this in the first place (the only place where we use strongly type

Reply:
AFAIK the only place we we used `Date` type in our jackson powered model are in the aem module when serializing aem event